### PR TITLE
[[Bugfix 14887]] Only insert revInitialisationLibrary as an auxiliary stackfile if necessary

### PR DESCRIPTION
**Pull request by livecodefraser**

### CODE REVIEW

**Comment by livecodefraser** [comment]
Should fix the issue when building multiple standalones at the same time.

File: ide-support/revsaveasstandalone.livecodescript [hunk 1]
```diff
@@ -2173,8 +2173,12 @@ private command revBuildStandaloneDeploy pPlatform, pStackFilePath, pEnginePath,
       end if
    end repeat
    if sStandaloneSettingsA["auxiliary_stackfiles"] is not empty then
-      put the effective filename of stack "revInitialisationLibrary" before sStandaloneSettingsA["auxiliary_stackfiles"]
-      put "insert the script of stack revInitialisationLibrary into back" before sStandaloneSettingsA["startup_script"]
+      local tRevInitialisationLibrary
+      put the effective filename of stack "revInitialisationLibrary" into tRevInitialisationLibrary
+      if tRevInitialisationLibrary is not among the lines of sStandaloneSettingsA["auxiliary_stackfiles"] then
+         put tRevInitialisationLibrary & cr before sStandaloneSettingsA["auxiliary_stackfiles"]
+         put "insert the script of stack revInitialisationLibrary into back" & cr before sStandaloneSettingsA["startup_script"]
```
Comment (runrevmark) [inline ide-support/revsaveasstandalone.livecodescript]: I suggest quoting 'revInitialisationLibrary' here in the script string... Just to be on the safe-side :)

**Comment by runrevmark** [comment]
Reviewed-by: @runrevmark.

**Comment by runrevmark** [comment]
Pending tweak (fix wasn't quite right).

**Comment by runrevmark** [comment]
Reviewed-by: @runrevmark.

**Pull request merged**